data-prepper-plugins/s3-source/src/main/java/org/opensearch/dataprepper/plugins/source/configuration/SqsOptions.java

```diff
@@ -19,6 +19,9 @@ public class SqsOptions {
     private static final Duration DEFAULT_VISIBILITY_TIMEOUT_SECONDS = Duration.ofSeconds(30);
     private static final Duration DEFAULT_WAIT_TIME_SECONDS = Duration.ofSeconds(20);
     private static final Duration DEFAULT_POLL_DELAY_SECONDS = Duration.ofSeconds(0);
+    private static final Duration DEFAULT_MAX_BACKOFF = Duration.ofSeconds(600);
+    private static final Duration DEFAULT_BASE_DELAY = Duration.ofSeconds(30);
+    private static final int DEFAULT_MAX_RETRIES = 5;
```

Comment:
Based on my comment below about receiving messages in a loop, it might make sense to increase this value. Each time the max retries is reached, the loop will start again and so will the backoff. Thoughts?